projects/angular/src/utils/list-key-navigation/key-navigation-list.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { AfterViewInit, Directive, ElementRef, HostListener, Inject, Input } from '@angular/core';
+import {
+  focusElement,
+  getFlattenedFocusableItems,
+  initializeKeyListItems,
+  KeyNavigationCode,
+  setActiveKeyListItem,
+  validKeyNavigationCode,
+} from '@cds/core/internal';
```

Comment:
These imports cause problems for users even if they don't use the new directive. See https://vmware-clarity.slack.com/archives/C0JF8D2LB/p1655238275941749.